modules/rup/routes/prestacion.ts

```diff
@@ -692,13 +692,22 @@ EventCore.on('rup:prestacion:validate', async (prestacion: IPrestacionDoc) => {
 
         if (reg.elementoRUP) {
             const elemento = elementosRUPSet.getByID(reg.elementoRUP);
+
             if (elemento && elemento.dispatch) {
                 elemento.dispatch.forEach(hook => {
                     if (hook.method === 'validar-prestacion') {
                         EventCore.emitAsync(hook.event, { prestacion, registro: reg });
                     }
                 });
             }
+            if (elemento && elemento.tags) {
```

Comment:
Cada registro puede tener sus propios tags, de esta forma se pisa. El save iría después del for. 

Antes del for blanquearía todo los tags.

Reply:
listo